Problema ao compilar o sistema

Como a solução com Docker não funcionou, tentei seguir o passo-a-passo com virtualenv. Mas as instruções fornecidas também não funcionaram:

```
(pythonbrasil13) renne@capivara:pythonbrasil13-site$ make html serve
cd theme; make all
make[1]: Entering directory '/home/renne/projects/pythonbrasil13-site/theme'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/renne/projects/pythonbrasil13-site/theme'
pelican /home/renne/projects/pythonbrasil13-site/content -o /home/renne/projects/pythonbrasil13-site/output -s /home/renne/projects/pythonbrasil13-site/pelicanconf.py 
CRITICAL: KeyError: 'JINJA_EXTENSIONS'
Makefile:48: recipe for target 'html' failed
make: *** [html] Error 1
(pythonbrasil13) renne@capivara:pythonbrasil13-site$ 
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -10,7 +10,7 @@ markdown
 markupsafe
 mdx-video
 paramiko
-pelican==3.7
+pelican==3.6
 pillow==2.6.1
 pycrypto
 pygments

